src/views/virtualmachines/creation-wizard-new/steps/InstanceTypesSteps/BootSourceStep/components/BootableVolumeList/components/BootableVolumeRow/BootableVolumeRow.tsx

```
@@ -77,9 +81,17 @@ const BootableVolumeRow: FC<BootableVolumeRowProps> = ({
   const bootVolumeNamespace = getNamespace(bootableVolume);
   const sizeData = getHumanizedSize(getDiskSize(dvSource, pvcSource, volumeSnapshotSource)).string;
   const icon = getVolumeNameOSIcon(bootVolumeName) || getTemplateOSIcon(preference);
+  const { setValue, getValues } = useVMWizard();
```

ESLint

[error] Expected "getValues" to come before "setValue"

(perfectionist/sort-objects)